Packs/SplunkPy/ReleaseNotes/2_1_12.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### SplunkPy
+- Updated the Docker image to: *demisto/splunksdk:1.0.0.24033*.
+- Maintenance and stability enhancements.
```

Comment:
specify the changes made to the integration

Reply:
Fixed